source/validator/validator.cpp

```diff
@@ -151,21 +151,23 @@ int main(int argc, char** argv) {
     error_count += check_relations(d.pt_data->stop_points, &StopPoint::stop_area, d.pt_data->stop_areas);
 //    error_count += check_relations(d.pt_data->stop_points, &StopPoint::admin_list, d.geo_ref.admins);
     error_count += check_relations(d.pt_data->stop_points, &StopPoint::network, d.pt_data->networks);
-    error_count += check_relations(d.pt_data->stop_points, &StopPoint::journey_pattern_point_list, d.pt_data->journey_pattern_points);
-    error_count += check_relations(d.pt_data->stop_points, &StopPoint::journey_pattern_point_list, d.pt_data->journey_pattern_points, &JourneyPatternPoint::stop_point);
+    //error_count += check_relations(d.pt_data->stop_points, &StopPoint::journey_pattern_point_list, d.pt_data->journey_pattern_points);
```

Comment:
I've just commented these lines as I didn't correct the code to test the new links. I don't know if this code is really useful or used somewhere. So:
* or I let it like that;
* or I check the new relations and remove the others;
* or I remove this exe.